tutorials/colors/colormap-manipulation.py

```diff
@@ -255,6 +255,43 @@ def plot_linearmap(cdict):
 
 plot_examples([cmap1, cmap2])
 
+#############################################################################
+# .. _reversing-colormap:
+#
+# Reversing a colormap
+# ====================
+#
+# Any colormap can be reversed using `.Colormap.reversed`. For example:
+
+colors = ["#ffffcc", "#a1dab4", "#41b6c4", "#2c7fb8", "#253494"]
+my_cmap = ListedColormap(colors, name="my_cmap")
+
+my_cmap_r = my_cmap.reversed()
+
+plot_examples([my_cmap, my_cmap_r])
+# %%
+# This method reverses the parent colormap. If no name is passed in, it also
+# names the reversed map by appending ``_r`` to the parent colormap's name.
+# See :ref:`registering-colormap` for an example.
+
+##############################################################################
+# .. _registering-colormap:
+#
+# Registering a colormap
+# ======================
+#
+# Colormaps can be added to the `matplotlib.colormaps` list of named colormaps.
+# This allows the colormaps to be accessed by name in plotting functions:
+
+mpl.colormaps.register(cmap=my_cmap)
+mpl.colormaps.register(cmap=my_cmap_r)
+
+plot_examples(['my_cmap', 'my_cmap_r'])
```

Comment:
Is it worth considering a small explicit imshow example here? Passing in strings vs. objects into some magic function, is very subtle and does not tell the user, that they can use the string anywhere a colormap is expected.